engines/router/Makefile

```diff
@@ -97,9 +97,8 @@ swagger-ui:
 
 
 .PHONY: build-local-router-image
-build-local-router-image: vendor
-	$(MAKE) build-image DOCKER_REGISTRY=localhost:5000 OVERWRITE_VERSION=latest
-	docker push $(IMAGE_TAG)
+build-local-router-image: vendor build-image
+	docker push ${DOCKER_REGISTRY}/turing-router:${OVERWRITE_VERSION}
```

Comment:
i think this line can remain as `docker push $(IMAGE_TAG)` since the is using `image_tag`